### DISCUSSION TITLE
Date in unix timestamp format / Consumption of the car chart

### DISCUSSION THREAD

**Discussion by Ilpo55** [flobz/psa_car_controller#491] [Q&A]
Does anyone know why the date in Consumption of the car chart is in unix timestamp format?
(1675T = 1675000000)   Sun Jan 29 2023
![Unixtime_PSA](https://user-images.githubusercontent.com/120249692/220569723-4a3d565e-ef33-477d-9a07-e28a01c31317.jpg)

**Comment by mememel** [comment]
I Have the same problem for a while ??! Anyone can help us ?

**Reply by lvenditti** [reply]
Same issues.

Could depend on Python release ?
I am using raspberry PI 4 with:
Linux pi4 5.10.103-v7l+ #1529 SMP Tue Mar 8 12:24:00 GMT 2022 armv7l GNU/Linux
and 
Docker version 23.0.4, build f480fb1

**Comment by lvenditti** [comment]
No other users have time in epoch format?

Thanks.

**Reply by stevoh6** [reply]
I have it too.
